pkg/bmc/access.go

```diff
@@ -107,21 +109,21 @@ func getParsedURL(address string) (parsedURL *url.URL, err error) {
 
 // NewAccessDetails creates an AccessDetails structure from the URL
 // for a BMC.
-func NewAccessDetails(address string, disableCertificateVerification bool) (AccessDetails, error) {
+func NewAccessDetails(host *metal3v1alpha1.BareMetalHost) (AccessDetails, error) {
```

Comment:
We don't want this package to depend on the host API, because this package is used in the OpenShift installer where we don't have a host API.

Reply:
Ok, thanks, will revert the change